### ISSUE TITLE
Paused video resumes when dismissing share or related dialog

### ISSUE THREAD

**Issue by mattcoz** [jwplayer/jwplayer#953]
Pause a video
Click on share
Dismiss the share dialog
Video resumes

It should stay paused.

**Comment by donato** [comment]
Hi @mattcoz - we thought this was the most natural use-case, however if more people are interested in this we can re-look at it. For your solution, I'd suggest adding an event listener for when related is closed and pauses the video.

```js
    var p = jwplayer('my-player');
    var relatedPlugin = p.getPlugin('related')
    relatedPlugin.on('close', function() { p.pause(); });
```

**Comment by mattcoz** [comment]
That would pause the video even if it was playing, it should return to the state that it was previously in. I've tried to detect it using relatedPlugin.on('open', ...), but you've already paused it at that point. Here is some code I ended up using that works:

```javascript
player.on('ready', function(data){
	var state;
	player.getPlugin('related').on('close', function(data){
		if (state == 'paused') window.setTimeout(function(){
			player.pause();
		}, 1);
	});
	window.setTimeout(function(){
		document.getElementsByClassName('jw-related-dock-btn')[0].onmousedown = function(){
			state = player.getState();
		};
	}, 1);
});
```

Kinda ugly.

**Comment by donato** [comment]
True, thanks for giving it a shot. I demo'ed internally the changes necessary for sharing/related to work how you suggest. 

This could be out as soon as v7.3.0 - assuming that we decide staying paused is a better user-experience. 

If we decide not to do that, I would be happy to improve the API so this won't be such a pain, for example the close event could give some extra info allowing:

```js
player.on('ready', function(data){
    player.getPlugin('related').on('close', function(obj){
        if (obj.stateWhenOpened == 'paused') {
            player.pause();
        }
    });
```

**Comment by mattcoz** [comment]
Cool, thanks.

**Comment by robwalch** [comment]
This issue has been resolved in v7.3.0
